ui/config_loader.py

```diff
@@ -1,9 +1,10 @@
 import os.path
 from ConfigParser import ConfigParser
 
-def load ():
+config_file = 'config.rc'
+def load (config_file = config_file):
```

Comment:
I did it initially so an external program could use the same function from a different dir. Then I changed my mind but left it in in case it's useful later.